src/app/pipes/shared/ksi-date.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import {DatePipe} from '@angular/common';
+
+@Pipe({
+  name: 'ksiDate'
+})
+export class KsiDatePipe implements PipeTransform {
+
+  transform(value: string, ...args: any[]): string | null {
```

Comment:
Navrhuji použít raději `never` než `any` - `any` totiž ničí typovou kontrolu, kdežto `never` říká, že nám osobně nezáleží na tom, co tam je (a kdybychom to chtěli někdy použít, tak nás to upozorní, že neznáme typ)
```suggestion
  transform(value: string, ...args: never[]): string | null {
```